spec/support/precomple_assets.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.configure do |config|
```

Comment:
Credit to Evil Martians [system test guide](https://evilmartians.com/chronicles/system-of-a-test-setting-up-end-to-end-rails-testing) for this.